### PR TITLE
Added unit tests for the ios code.

### PR DESCRIPTION
**Pull request by gaaclarke**

### CODE REVIEW

**Comment by liyuqian** [comment]
I'm curious why we're adding third party code inside the engine repo? Should all third party code live inside src/third_party instead of src/flutter/third_party ?

**Comment by gaaclarke** [comment]
@chinmaygarde Done.

File: DEPS [hunk 1]
```diff
@@ -376,6 +380,9 @@ deps = {
   'src/third_party/skia':
    Var('skia_git') + '/skia.git' + '@' +  Var('skia_revision'),
 
+  'src/third_party/ocmock':
+   Var('ocmock_git') + '@' +  Var('ocmock_tag'),
```
Comment (chinmaygarde) [inline DEPS]: Nit: The convention is `Var('github_git') + 'erikdoe/ocmock.git@' ...`

**Comment by liyuqian** [comment]
@gaaclarke : is this test currently running in our Cirrus pre-submit tests, or LUCI post-submit tests?

**Comment by gaaclarke** [comment]
@liyuqian Neither, I'd like to set it up.  Since it has to run on a mac I'm not sure what that means for automating the tests.  Can you point me in the right direction to get them running automatically?

**Comment by liyuqian** [comment]
Mac on Cirrus is too slow so we only run Mac tests on LUCI. I'll message you privately about the LUCI doc as I'm not sure whether I should link internal doc here.

**Pull request merged**